testdata/dnn/download_models.py

```diff
@@ -69,12 +69,18 @@ def get(self):
 
         print(' done')
         print(' file {}'.format(self.filename))
-        return self.verify()
+        try:
```

Comment:
It is better to move this `try .. catch .. ` into `def verify(self)` and catch only specific exceptions - file can no be open or read.